### PR TITLE
add syntax coloring.

### LINKED ISSUE #656: add syntax coloring to docs and blog.
We should add syntax highlighting to code snippets as we had on our old site. 

image

### PR DESCRIPTION
**Pull request by nisarhassan12**
Fixes #656 

<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/670"></a>

### CODE REVIEW

File: src/routes/docs/self-hosted/latest/install/database.md [hunk 1]
```diff
@@ -19,15 +19,15 @@ Gitpod uses a MySQL database to store user data. By default Gitpod ships with a
 This chart installs a MySQL database that gets Gitpod up and running but is not suitable for production (the data is lost on each restart of the DB pod). To connect to a proper MySQL installation:
 
 1.  Copy the DB init scripts into your local folder:
-    ```
+    ```bash
```
Comment (nisarhassan12) [inline src/routes/docs/self-hosted/latest/install/database.md:22]: If we don't specify the language it does not work properly:

image

**Comment by mikenikles** [comment]
This should be supported out of the box by mdsvex ([docs](https://mdsvex.com/docs#highlight)). Let's not add external CSS and JS files to accomplish this, but rather investigate why syntax highlighting currently doesn't work in our case.

A related conversation is open at https://github.com/pngwn/MDsveX/issues/224.

**Comment by nisarhassan12** [comment]
Thanks. @mikenikles I looked into it and the reason why it was not working was not having the css for any of the prism theme as mention in the [docs](https://mdsvex.com/docs#highlight). 

Also I just forced pushed after nuking the changes that were no longer relevant. 

It now seems to work perfectly but we have to make sure that we have the lang attribute specified in all places in order for it to work. So far I have found one place now looking for other places and specifying the relevant languages.

File: static/prism-solarized-light.css [hunk 2]
```diff
@@ -0,0 +1,155 @@
+/* PrismJS 1.23.0
+https://prismjs.com/download.html#themes=prism-solarizedlight&languages=markup+css+clike+javascript */
```
Comment (mikenikles) [inline static/prism-solarized-light.css:2]: Could we please use https://prismjs.com/download.html#themes=prism-solarizedlight&languages=markup+css+clike+javascript+bash+docker+yaml and make sure we use the minified version?

**Review by mikenikles — COMMENTED** [review]
Thank you Nisar, sometimes it's the small things that make a big difference - such as adding a CSS file 😅.

File: static/prism-solarized-light.css (hunk 2 above)
Comment (nisarhassan12) [reply static/prism-solarized-light.css:2]: Thanks. @mikenikles. I was expecting sveltekit to optimize the css and js assests in the static directory but it turns out that this is not what is happening. 

Now I have switched to use the minified version.

**Comment by mikenikles** [comment]
Wow, nice work updating all code snippets to make syntax highlighting work. It even surfaced a bunch of indentation bugs in yaml code snippets 🤯.

The PR is in draft, I'll merge it once it's ready for review.

**Comment by nisarhassan12** [comment]
Thanks. @mikenikles I have now marked it ready for review Sorry I didn't do that earlier.

File: static/prism-solarized-light.css [hunk 3]
```diff
@@ -0,0 +1,103 @@
+code[class*="language-"],
+pre[class*="language-"] {
+  color: #657b83;
+  font-family: Consolas, Monaco, "Andale Mono", "Ubuntu Mono", monospace;
+  font-size: 1em;
+  text-align: left;
+  white-space: pre;
+  word-spacing: normal;
+  word-break: normal;
+  word-wrap: normal;
```
Comment (nisarhassan12) [inline static/prism-solarized-light.css:10]: Looks like the pre commit hook is adding the spaces and newlines which were added during the minification. @mikenikles Is that ok ? Or should I look into How we can ignore this file ?

File: src/routes/blog/native-ui-with-vnc.md [hunk 4]
```diff
@@ -23,10 +23,12 @@ A few days ago [Gero introduced Docker builds to Gitpod](/blog/docker-in-gitpod)
 
 Let’s use Gitpod to write <a class="no-nowrap" href="https://github.com/csweichel/gitpod-hello-ui-demo">a simple desktop-native application in Go</a> using [libui](https://github.com/andlabs/libui). First, we create a GitHub repo and a [little bit of configuration](https://github.com/csweichel/gitpod-hello-ui-demo/commit/fea580735c09fa704531a810e8ec7ca6a5c03a88): we need a Dockerfile to install libui’s dependencies (see below), and we need to tell Gitpod to use that Dockerfile.
 
-    FROM gitpod/workspace-full-vnc
-    RUN sudo apt-get update && \
-        sudo apt-get install -y libgtk-3-dev && \
-        sudo rm -rf /var/lib/apt/lists/*
+```dockerfile
+FROM gitpod/workspace-full-vnc
+RUN sudo apt-get update && \\
```
Comment (nisarhassan12) [inline src/routes/blog/native-ui-with-vnc.md:28]: @mikenikles I have added the second slash to fix the newline issue where the slash was not showing up. Once it is fixed upstream and we make an upgrade we should make sure that we remove the 2nd slash. Should I open an issue for it ? Thanks

Comment (mikenikles) [reply src/routes/blog/native-ui-with-vnc.md:28]: No need, we already have #98 😉

File: static/prism-solarized-light.css (hunk 3 above)
Comment (mikenikles) [reply static/prism-solarized-light.css:10]: Ah let me look into that. I'll see if I can rename the file to `*.min.css` and add an exclusion rule to Prettier.

**Pull request merged**